== Function called ==
Value to be returned:
    int(5)

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got integer.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got integer.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    array(1) {
      [0]=>
      int(5)
    }

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got array.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got array.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    string(6) "string"

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got string.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got string.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    NULL

.test {
    width: 5px;
}



== Function called ==
Value to be returned:
    bool(true)

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got boolean.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got boolean.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    object(stdClass)#2 (0) {
    }

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got instance of stdClass.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got instance of stdClass.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    object(SassFakeValue)#3 (0) {
    }

Caught exception 'Sass\SassException' with message 'error in callback function func: Expected instance of Sass\Types\SassValue, got instance of SassFakeValue.' in %s/tests/customfunctions/returnvalues.php:25
Source File: stdin
Source Line: 1
Source Column: 31
Full message:
Error: error in callback function func: Expected instance of Sass\Types\SassValue, got instance of SassFakeValue.

       Backtrace:
       	stdin:1, in function `func`
       	stdin:1
        on line 1 of stdin
>> .test { width: 5px; content: func(); }
   ------------------------------^

Stack trace:
#0 %s/tests/customfunctions/returnvalues.php(25): Sass\Sass->compile()
#1 %s/tests/customfunctions/returnvalues.php(45): runFunction()
#2 %s/tests/customfunctions/returnvalues.php(49): testCFReturnValues()
#3 {main}


== Function called ==
Value to be returned:
    object(Sass\Types\SassString)#4 (2) {
      ["value":"Sass\Types\SassString":private]=>
      string(9) "sassvalue"
      ["isQuoted":"Sass\Types\SassString":private]=>
      bool(false)
    }

.test {
    width: 5px;
    content: sassvalue;
}



